db/schema.rb

```diff
@@ -13,7 +13,7 @@
 ActiveRecord::Schema.define(version: 2022_07_25_163014) do
 
   create_table "barcodes", force: :cascade do |t|
-    t.integer "product_id"
+    t.integer "product_id", null: false
```

Comment:
These changes break the tests. We probably need to update the factories in the tests (and/or database seed).

Reply:
Fixed in #195.